Google.Api.Generator/Generation/SnippetCodeGenerator.cs

```diff
@@ -726,7 +762,34 @@ public class Signature
 
                 private string SyncMethodName => Method.SyncMethodName + OverloadDisambiguation;
                 private string AsyncMethodName => $"{Method.AsyncMethodName.Substring(0, Method.AsyncMethodName.Length - 5)}{OverloadDisambiguation}Async";
-                
+
+                private IEnumerable<ClientMethod.Types.Parameter> CoreClientMethodParameters => _sig.Fields.Select(
+                    field => new ClientMethod.Types.Parameter
```

Comment:
It might be worth separating out helper methods of `CreateParameter(Type, string)` and `CreateParameter(Typ, string)`.

Reply:
Yep, done! Will push again and merge on green. Thanks!